### PR TITLE
braincurses: init at 1.1.0

### PR DESCRIPTION
**Pull request by dotlambda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/games/braincurses/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    repo = "braincurses";
+    rev = version;
+    sha256 = "0gpny9wrb0zj3lr7iarlgn9j4367awj09v3hhxz9r9a6yhk4anf5";
+  };
+
+  buildInputs = [ ncurses ];
+
+  # There is no install target in the Makefile
+  installPhase = ''
+    install -D braincurses $out/bin/braincurses
```
Comment (yegortimoshenko) [inline pkgs/games/braincurses/default.nix]: Could be even shorter :-)

```
install -Dt $out/bin braincurses
```

**Review by lukateras — APPROVED** [review]
I'm not sure how to play it, but it works. All good, ready to merge!

File: pkgs/games/braincurses/default.nix (hunk 1 above)
Comment (dotlambda) [reply pkgs/games/braincurses/default.nix]: Thanks for the hint! I've shortened it.

**Review by c0bw3b — APPROVED** [review]
Builds and runs

**Comment by dotlambda** [comment]
@GrahamcOfBorg build braincurses

**Comment by c0bw3b** [comment]
It won't because of `platforms.linux`

**Comment by grahamc** [comment]
@GrahamcOfBorg build braincurses

**Comment by yegortimoshenko** [comment]
@dotlambda Try to build it with `nix-build -E "with import <nixpkgs> {}; callPackage ./. { stdenv = clangStdenv; }". If it does, there's a good chance it will build on Darwin.

**Comment by dotlambda** [comment]
@yegortimoshenko It does. Do you think I should set `platforms` to `unix`? Also, do you know of any easy way to test on macOS without Apple hardware?

**Pull request merged**